### PR TITLE
Release 4.4.0: Update demo apps

### PR DESCRIPTION
**Pull request by DmitriyKirakosyan**
Merge changes in Puppet apps into Demo apps.

[AB#87998](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/87998)

### CODE REVIEW

File: Apps/Contoso.UWP.Demo/App.xaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿<Application
-    x:Class="Contoso.UWP.Demo.App"
+    x:Class="Contoso.UWP.Puppet.App"
```
Comment (AnastasiaKubova) [inline Apps/Contoso.UWP.Demo/App.xaml:2]: Revert please these changes

File: Apps/Contoso.UWP.Demo/App.xaml.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@
 using Microsoft.AppCenter.Crashes;
 using System.Threading.Tasks;
 
-namespace Contoso.UWP.Demo
+namespace Contoso.UWP.Puppet
```
Comment (AnastasiaKubova) [inline Apps/Contoso.UWP.Demo/App.xaml.cs:17]: Revert please these changes

File: Apps/Contoso.UWP.Demo/App.xaml.cs [hunk 3]
```diff
@@ -27,16 +27,22 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
-             TaskScheduler.UnobservedTaskException += (object sender, UnobservedTaskExceptionEventArgs args) =>
+            TaskScheduler.UnobservedTaskException += (object sender, UnobservedTaskExceptionEventArgs args) =>
             {
                 // If you see this message while testing the app and if the stack trace is SDK related, we might have a bug in the SDK as we don't want to leak any exception from the SDK.
-                AppCenterLog.Error("AppCenterDemo", "Unobserved exception observed=" + args.Observed, args.Exception);
+                AppCenterLog.Error("AppCenterPuppet", "Unobserved exception observed=" + args.Observed, args.Exception);
```
Comment (AnastasiaKubova) [inline Apps/Contoso.UWP.Demo/App.xaml.cs:33]: Revert please these changes

File: Apps/Contoso.UWP.Demo/App.xaml.cs [hunk 4]
```diff
@@ -27,16 +27,22 @@ sealed partial class App : Application
...
             Suspending += OnSuspending;
             AppCenter.LogLevel = LogLevel.Verbose;
-            AppCenter.Start("e8354a9a-001a-4728-be65-a6477e57f2e7", typeof(Analytics), typeof(Crashes));
+            AppCenter.SetLogUrl("https://in-integration.dev.avalanch.es");
+            object storageSize;
+            if (Windows.Storage.ApplicationData.Current.LocalSettings.Values.TryGetValue("StorageMaxSize", out storageSize))
+            {
+                AppCenter.SetMaxStorageSizeAsync((long)storageSize);
+            }
+            AppCenter.Start("42f4a839-c54c-44da-8072-a2f2a61751b2", typeof(Analytics), typeof(Crashes));
```
Comment (AnastasiaKubova) [inline Apps/Contoso.UWP.Demo/App.xaml.cs:45]: Revert please these changes

File: Apps/Contoso.UWP.Demo/App.xaml.cs [hunk 5]
```diff
@@ -97,6 +103,7 @@ void OnNavigationFailed(object sender, NavigationFailedEventArgs e)
             throw new Exception("Failed to load Page " + e.SourcePageType.FullName);
         }
 
+
```
Comment (AnastasiaKubova) [inline Apps/Contoso.UWP.Demo/App.xaml.cs:106]: Extra line

Comment (DmitriyKirakosyan) [reply Apps/Contoso.UWP.Demo/App.xaml.cs:105]:
```suggestion
```

**Review by AnastasiaKubova — APPROVED** [review]

**Pull request merged**